doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml

```diff
@@ -37,5 +37,12 @@
             <summary>Logs information through a specified method of the current instance type.</summary>
             <remarks>To be added.</remarks>
         </LogInfo>
+        <LogWarning>
+          <param name="type">The type to be logged.</param>
+          <param name="method">The logging method.</param>
+          <param name="message">The message to be logged.</param>
+          <summary>Logs warning through a specified method of the current instance type.</summary>
+          <remarks>To be added.</remarks>
```

Comment:
Avoid adding "To be added" for new documentations.. 
Either provide remarks or remove this XML element.